README.md

```diff
@@ -1,208 +1,262 @@
+      ]
+   },
+   "cachebust":"635689412670032000"
 }
 ```
-For more information on components file please read this documentation:
-[Mendix Components](https://world.mendix.com/display/refguide5/Customizing+Hybrid+Mobile+Apps)
 
-The components file should be included in the theme folder of your application.
+### Step 5 - Update index.html file
 
+Update `theme/index.html`to include the following reference to this javascript library:
 
-## Sending push notifications
+```
+<script type="text/javascript" src="widgets/pushNotifications/lib/PushNotification.js"></script>
+```
 
-In the module there are two operations to send messages. One which will queue the messages and the other to send immediately. If you are planning to send several push notifications to different devices at once, then I would suggest using the queued method. If however you simply want to test the sending of messages then just use the send immediate microflow.
+<img src="assets/images/implementation guide/Update index html file.PNG"/>
 
-In the module there is a devices page. This will list all of the devices that are currently registered to the application. If you want to send a message to one of these devices you can simply select a device and click send message.
+### Step 6 - Include Push Notification Snippet in the layouts
 
-To send a message without using the devices page, simply create a microflow that retrieves the device from a user account and create a message object with the attributes filled. Then simply pass this message to one of the microflows in the use me folder that sends either a message or a list of messages.  
+Include a push notification snippet on mobile and tablet layouts. You can do this by drag-and-dropping the `PushNotification_Snippet` snippet (located in `_USE ME/Snippets` folder in PushNotifications module) to your layout.
 
-## Setting up Apple Push Notification Server
-In order to send push notifications for apple devices from this module you will need to correctly set up and aquire a certificate from apple, then add this to the Mendix settings pages.
+<img src="assets/images/implementation guide/Include push notification snippet on layouts.JPG"/>
 
-### Step 1 - Login to members center
-Login to the [members center] (https://developer.apple.com/) on developer.apple.com. If you do not have an apple developers license you will need to purchase this from apple. When logged in to members center click on "Certificates, Identifiers & Profiles". 
+### Step 7 - Set up the administration pages
 
-This will take you to a screen like this:
-<img src="assets/images/apn-step1.PNG"/>
+Add `AppleAdministration`, `GoogleAdministration`, and `Device_Overview` pages to the project navigation. The `Apple Administration` and `GoogleAdministration` pages are used to configure your application to be able to reach the respective services (APNs and GCM) later on. The `Device_Overview` page is useful for testing purpose.
```

Comment:
'Add *the*  pages'

Maybe add 'The `Device_Overview` page contains a list of all devices registered to the application, and is useful for testing purposes.' (purposes should be plural either way imo)